src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const indexToCopy = 1;
+const indexToInsert = table.rows[indexToCopy].cells.length - 1;
+
+for (let i = 0; i < table.rows.length; i++) {
+  const tagToCopy = table.rows[i].cells[indexToCopy].tagName;
+
+  const copiedElement = document.createElement(tagToCopy);
+
+  copiedElement.textContent = `${table.rows[i].cells[indexToCopy].innerText}`;
+
+  table.rows[i].insertBefore(copiedElement, table.rows[i].cells[indexToInsert]);
+}
```

Comment:
let's try instead of creating a completely new element make a cell copy (read about Node.cloneNode) and insert it
hint: to do this you need to find all table row elements in form and while iterating through each of them insert a copied cell to the right place (you can use insertBefore for this as well)